controllers/csiscaleoperator_controller.go

```diff
@@ -912,8 +916,18 @@ func (r *CSIScaleOperatorReconciler) updateStatus(instance *csiscaleoperator.CSI
 }
 
 func (r *CSIScaleOperatorReconciler) isControllerReady(controller *appsv1.Deployment) bool {
-	return controller.Status.ReadyReplicas == MinControllerReplicas
-
+	logger := csiLog.WithName("isControllerReady")
+	logMessage := "Controller status"
+	logKey := "ReadyReplicas == MinControllerReplicas"
+	logValueTrue := "True"
+	logValueFalse := "False"
+	if controller.Status.ReadyReplicas == MinControllerReplicas {
+		logger.Info(logMessage+":", logKey+":", logValueTrue)
+		return true
+	} else {
+		logger.Info(logMessage+":", logKey+":", logValueFalse)
```

Comment:
We can log value of controller.Status.ReadyReplicas here.

Reply:
Done, added a log statement here.